app/controllers/gardens_controller.rb

```diff
@@ -16,26 +19,25 @@ def show
   def new; end
 
   def create
-    # POST "api/v1/gardens" to create a garden using strong params
-    # note pass in current_user.id into hash for back-end association
-    redirect_to dashboard_path
+    GardenFacade.new_garden(garden_params, current_user.id)
+    redirect_to gardens_path
   end
 
   def edit
     # GET "api/v1/gardens/params[:id]" to obtain garden from id
 
               # this code is used only for testing
-    @garden = Garden.new(current_user.gardens.first)
+  @garden = Garden.new(current_user.gardens.first)
   end
 
   def update
     # PATCH "api/v1/gardens/params[:id]" to update the garden using strong params
-    redirect_to dashboard_path
+    redirect_to gardens_path
   end
 
   def destroy
     # DELETE api/v1/gardens/:id' to destroy garden
-    redirect_to dashboard_path
+    redirect_back(fallback_location: dashboard_path)
```

Comment:
Used fallback location as user can also delete directly from dashboard and should remain on dashboard. When deleting from gardens index, they should remain on gardens index.

Reply:
Nice! Way to think about UX here.